docs/src/operate/c8y/configuration-management.md

```diff
@@ -26,13 +26,18 @@ files = [
   { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf', type = 'c8y-bridge.conf' },
   { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf', type = 'tedge-mosquitto.conf' },
   { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto.conf' },
-  { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o644 }
+  { path = '/etc/tedge/c8y/example.txt', type = 'example', user = 'tedge', group = 'tedge', mode = 0o644 },
+  { path = '/etc/containers/certs.d/example/ca.crt', type = 'harbor-certificate', user = 'tedge', group = 'tedge', mode = 0o640, parent_user = 'root', parent_group = 'root', parent_mode = 0o755 },
 ]
 ```
 
 * `path` is the full path to the configuration file.
 * `type` is a unique alias for each file entry which will be used to represent that file in Cumulocity UI.
-* `user`, `group` and `mode` are UNIX file permission settings to be used to create a configuration file. If not provided, the files will be created with `root` user. If the file exists already, its ownership will be retained.
+* `user`, `group` and `mode` are UNIX file permission settings to be used to create a configuration file. If not provided, the files will be created with the current user and group, or `root`.
```

Comment:
This sentence is not clear: "with the current user and group, or `root`". Which one is actually use?

Reply:
First of all, I would note that this change was made as an update based on the current implementation, regardless of this PR.

If the first attempt to create a file is successful, the current user and group are used. 
https://github.com/thin-edge/thin-edge.io/blob/main/crates/extensions/tedge_config_manager/src/actor.rs#L413

If it's failed, `tedge-write` will be called with `sudo` when it's enabled by the config. Then, the second attempt to create a file by `tedge-write` will use `root`.

Reply:
> If not provided, the files will be created with the current user

Is the current user the current user of the agent? I.e. `tedge`?

Reply:
Updated to reflect the current decision. b42bb22fb30b099dfd62868031af710c0cca1055
(I might have explained too much details about the ownership as "Getting Started"...)